modules/extensions/src/utils/syntax-tags.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Marks GLSL shaders for syntax highlighting: glsl`...`
+ * Install https://marketplace.visualstudio.com/items?itemName=boyswan.glsl-literal
+ */
```

Comment:
This is assuming the developer is using VSCode. FWIW I also got it to work using the following in vim:

```vim
" in .vimrc
autocmd Syntax * call SyntaxRange#Include('glsl`', '`', 'glsl', 'NonText')

```
with [vim-glsl](https://github.com/tikhomirov/vim-glsl) and [vim-SyntaxRange](https://github.com/inkarkat/vim-SyntaxRange)

I think it would be nice to also include here